packages/contracts-bedrock/test/ProtocolVersions.t.sol

```diff
@@ -71,7 +55,8 @@ contract ProtocolVersions_Initialize_Test is ProtocolVersions_Init {
         vm.expectEmit(true, true, true, true, address(protocolVersions));
         emit ConfigUpdate(0, ProtocolVersions.UpdateType.RECOMMENDED_PROTOCOL_VERSION, abi.encode(recommended));
 
-        vm.prank(multisig);
+        address admin = address(uint160(uint256(vm.load(address(protocolVersions), Constants.PROXY_OWNER_ADDRESS))));
+        vm.prank(admin);
```

Comment:
This should migrate to using `        excludeSender(EIP1967Helper.getAdmin(address(optimismPortal)));`